src/expensePoints.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+function expensePoints(app, { users }, { expenses }) {
+  let newExpenseID = 1;
+
+  app.get('/', (req, res) => {
+    const { userId, category, from, to } = req.query;
+
+    let getExpenses = [...expenses];
```

Comment:
getExpenses sounds like function. But it is a variable. MB it is better to call it receivedExpenses.. Smth like that